tinygrad/codegen/gpu.py

```diff
@@ -142,13 +142,12 @@ def load(self, buf_index:int, idx_override:Optional[str]=None) -> List[Token]:
   def ast_parse(self, x, acc:List[Token], do_reduce=False) -> List[Token]:
     if not isinstance(x, LazyOp): return self.load(self.bufs.index(x), "mid" if x is None else None)  # hack for local
     if isinstance(x.op, ReduceOps) and not do_reduce: return acc
-    values : List[List[Token]] = ([acc] if isinstance(x.op, ReduceOps) else []) + [self.ast_parse(v, acc, do_reduce) for v in x.src]
+    values: List[List[Token]] = ([acc] if isinstance(x.op, ReduceOps) else []) + [self.ast_parse(v, acc, do_reduce) for v in x.src]
     code = GPUCodegen.code_for_op[x.op]  # TODO: replace this with a function
-    if len(values) == 2:
```

Comment:
?

Reply:
This doesn't change the behaviour, I swear.

here is the change   
before :
```python
if len(values) == 2:
	return "behaviour len(values) == 2"
else:
	return "behaviour lend(values) != 2"
```	
	
Now :
```python
if len(values) != 2: return "behaviour lend(values) != 2"
return "behaviour len(values) == 2"
```
If you are afraid those changes can affect the proper functioning of tinygrad for no real added value I can ditch the format commit and only keep the flake E203 one.

Sorry for the inconvenience.